pennylane/math/single_dispatch.py

```diff
@@ -87,6 +87,14 @@ def _cond(pred, true_fn, false_fn, args):
 ar.register_function("numpy", "cond", _cond)
 ar.register_function("builtins", "cond", _cond)
 
+ar.register_function("numpy", "gamma", lambda x: _i("scipy").special.gamma(x))
+ar.register_function("numpy", "gammainc", lambda x, y: _i("scipy").special.gammainc(y, x))
+
+ar.register_function("builtins", "gamma", lambda x: _i("autograd.scipy").special.gamma(x))
+ar.register_function(

```

Comment:
I think it might be better to register gammainc in multi dispatch, because potentially with two float you end up with an autograd object. Generally the first object is always a float and the second one can be autograd or jax right?

Reply:
I changed the order of the parameters so that the first object is always autograd/jax object. Everything works this way but can bring the function to multi dispatch if it is a better option.